plugins/modules/mso_schema_template_vrf.py

```diff
@@ -46,6 +47,15 @@
     description:
     - Whether to enable vzAny.
     type: bool
+  ip_data_plane_learning:
+    description:
+    - Whether IP data plane learning is enabled or disabled.
```

Comment:
Is there a default when not provided? If so should this be added to the description?

Reply:
The default value set when you don't send it. which is why we do not need default.
https://github.com/CiscoDevNet/ansible-mso/pull/358#discussion_r1157651052

Reply:
Correct this is what I thought, but then we document in the description.
```suggestion
    - Whether IP data plane learning is enabled or disabled.
    - The APIC defaults to C(enabled) when unset during creation.
```

Reply:
Whether IP data plane learning is enabled or disabled. -> 
Allows setting IP data plane learning as enabled or disabled

Reply:
Since all the previous arguments are explained that way, changing explanation for one argument is not feasible.